src/main/java/io/github/thebusybiscuit/slimefun4/utils/UnbreakingAlgorithms.java

```diff
@@ -0,0 +1,15 @@
+package io.github.thebusybiscuit.slimefun4.utils;
+
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.DamageableItem;
+
+/**
+ * This a enum indicating if a {@link DamageableItem} is a tool or an armor
+ * This effects the way they determine if an item should be damaged
+ *
+ * @author RobotHanzo
+ */
+public enum UnbreakingAlgorithms {
```

Comment:
Also enums should be in singular, not plural form.

```suggestion
public enum UnbreakingAlgorithm {
```